ipatests/test_integration/test_trust.py

```diff
@@ -268,6 +259,65 @@ def test_override_homedir(self):
             tasks.restore_files(self.master)
             tasks.clear_sssd_cache(self.master)
 
+    def test_extdom_plugin(self):
+        """Extdom plugin should not return error (32)/'No such object'
+
+        Regression test for https://pagure.io/freeipa/issue/8044
+
+        If there is a timeout during a request to SSSD the extdom plugin
+        should not return error 'No such object' and the existing user should
+        not be added to negative cache on the client.
+        """
+        client = self.clients[0]
+        tasks.backup_file(self.master, paths.SSSD_CONF)
+        log_file = '{0}/sssd_{1}.log' .format(paths.VAR_LOG_SSSD_DIR,
+                                              self.master.domain.name)
+        logsize = len(self.master.get_file_contents(log_file))
+        res = self.master.run_command(['pidof', 'sssd_be'])
+        pid = res.stdout_text.strip()
+        try:
+            test_id = 'id testuser@%s' % self.ad_domain
+            self.master.run_command(test_id)
+            tasks.kinit_admin(self.master)
```

Comment:
I am not quite sure but it looks useless as you are not executing any `ipa` commands.